dts/bindings/base/mutable.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023, Meta
+# SPDX-License-Identifier: Apache-2.0
+
+# Properties for Mutable devices (PM)
+
+properties:
+  zephyr,mutable:
+    type: boolean
+    description: |
+      True iff the device structure may be mutated.
```

Comment:
Small typo

Reply:
> Small typo

"Iff" is a mathematical term meaning "if and only if"

Reply:
okay, did not know that :^)